internal/gen/controller.swagger.json

```diff
@@ -3729,6 +3729,33 @@
         ]
       }
     },
+    "/v1/workers/:create:worker-led": {
```

Comment:
This path changed 🤔

Reply:
This change appears to be limited to the openapiv2 generator, the handler is still doing the same thing.

Reply:
I'm pretty sure this is caused by https://github.com/grpc-ecosystem/grpc-gateway/pull/2727

Reply:
Can we wait to merge this until that PR is pulled in too?

Reply:
I've got a fix in https://github.com/grpc-ecosystem/grpc-gateway/pull/2825.

Reply:
I can confirm that this PR fixes the issue for us. I would like to wait for one of the other gateway contributors to comment on it before merging though.